tests/by-util/test_seq.rs

```diff
@@ -176,3 +176,26 @@ fn test_width_negative_zero() {
         .stdout_is("-0\n01\n")
         .no_stderr();
 }
+
+#[test]
+fn test_ignore_leading_whitespace() {
+    new_ucmd!()
+        .arg("   1")
+        .succeeds()
+        .stdout_is("1\n")
+        .no_stderr();
+}
+
+#[test]
+fn test_trailing_whitespace_error() {
+    new_ucmd!()
+        .arg("1 ")
+        .fails()
+        .no_stdout()
+        // FIXME GNU error message has curly quotes (‘) instead of
```

Comment:
I think this depends on your locale. If you run `LC_ALL=C seq "1  "` (i.e. use the `C` locale) it prints `seq: invalid floating point argument: '1  '`, so I think the current behavior is correct because we don't support other locales right now.

Reply:
I see. I'll change this comment to be less of a "FIXME" and more of a "something to watch out for".